installer/scripts/omsadmin.sh

```diff
@@ -352,6 +367,25 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
Customer would not know what is the minimum accepted limit where the agent can still function.